files/r11k.sh

```diff
@@ -208,7 +212,12 @@ function do_submodules() {
 	done
 }
 function translate_branch_to_env() {
-	echo -n "$1" | perl -pe 's@/@__@g;s@([^a-zA-Z0-9_])@sprintf "_%02x", ord($1)@ge;'
+    if [ -n "${R11K_PRODUCTION_BRANCH}" -a "${1}" == "${R11K_PRODUCTION_BRANCH}" ]
```

Comment:
en wat als der ook een production branch bestaat? mss dubbel checken als production dan niet de r11k_production_branch is...